components/cellery-component-test/src/test/java/org/cellery/components/test/EmployeeTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Please update the year in the license header. Please fix in the other files as well

Reply:
Done